client/src/redux/store.js

```diff
@@ -1,7 +1,8 @@
 import { configureStore } from "@reduxjs/toolkit";
+import authReducer from "./auth-slice";
 
 export const store=configureStore({
     reducer:{
-        // reducer
+        authReducer
     }
 })
```

Comment:
Since this is a frontend issue, It is generally considered a good practice to have video proofs available for whatever feature you are implementing. Please update the PR accordingly